doc/lsst.ap.verify/datasets-creation.rst

```diff
@@ -60,18 +58,14 @@ The dataset's package-level documentation should include:
 
 .. _ap-verify-datasets-creation-config:
 
-Configuring dataset ingestion and use
-=====================================
+Configuring dataset use
+=======================
 
-Each dataset's :file:`config` directory should contain a :ref:`task config file<command-line-task-config-howto-configfile>` named :file:`datasetIngest.py`, which specifies a `DatasetIngestConfig`.
-The file typically contains filenames or file patterns specific to the dataset.
-In particular, the default config ignores reference catalogs, so the config file should provide a ``dict`` from catalog names to their tar files.
-
-Each :file:`config` directory may contain a task config file named :file:`apPipe.py`, specifying an `lsst.ap.pipe.ApPipeConfig`.
-The file contains pipeline flags specific to the dataset, such as the available reference catalogs (both their names and configuration) or the type of template provided to `~lsst.pipe.tasks.imageDifference.ImageDifferenceTask`.
+The files in :file:`config` or :file:`pipelines` should :ref:`override any config fields<pipeline_creating_config>` that are constrained by the input data, such as template type or refcat filters, even if the current defaults match.
```

Comment:
> Does this mean that one should override all default settings?

Only those that specify what the pipeline inputs are (e.g., if you try to difference using good-seeing templates when you only have deep templates, or vice versa, the pipeline simply won't run). This is as opposed to things like "number of fitting iterations", where there isn't Only One Right Answer and it's more a matter of best practice.

> Do you have an example of an appropriate... config?

The two I mentioned (good seeing vs. deep, or refcat configuration) would be good examples if we could be consistent about what the defaults are in the first place...

Reply:
Thanks for the clarification. See if you can incorporate some text like that into the doc, please.

Reply:
Can you be more specific? I don't think I've said anything different in the comments from what I said in the docs...

Reply:
I think if you added something like "For example, it is a good idea to explicitly specify `config.coaddName="deep"` when that is the type of coadd included in the preloaded data, because that field is not consistent across different pipelines." or something like that. The "template type" wasn't specific enough for me.

Reply:
How about:
> The files in `config` or `pipelines` should override any config fields that are constrained by the input data, such as template type (deep, goodSeeing, etc.) or refcat filters, even if the current defaults match. This policy makes the datasets more self-contained and prevents them from breaking when the pipeline defaults change but only one value is valid (e.g., `coaddName` *must* be `"deep"` for a dataset with deep coaddds).

Reply:
That's very helpful, thank you!